src/main/java/tech/andreagreco/FileReaderTest.java

```diff
@@ -0,0 +1,20 @@
+package tech.andreagreco;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class FileReaderTest {
+    private void readFile(String fileName) throws IOException {
```

Comment:
Why not using a testing framework?
It's also not understandable what exactly are you testing